pandas/io/sql.py

```diff
@@ -963,7 +963,7 @@ def _sqlalchemy_type(self, col):
             TIMESTAMP,
         )
 
-        if col_type == "datetime64" or col_type == "datetime":
+        if col_type in ("datetime64", "datetime", "datetimetz"):
```

Comment:
can you revert all of these changes in this file

Reply:
I think we need this to pass the tests. As you can see there were some failures. I will try some tests locally (I have to copy paste them since I am not sure who to change the configuration for the connection strings) and give you a new feedback when I'm done :).